packages/fusion-runtime/src/executor.ts

```diff
@@ -70,9 +70,7 @@ export function getExecutorForUnifiedGraph<TContext>(
     enumerable: true,
     get() {
       return function unifiedGraphExecutorDispose() {
-        opts?.transportContext?.logger?.debug(
-          'Disposing unified graph executor',
-        );
+        opts?.transportContext.log.debug('Disposing unified graph executor');
```

Comment:
`transportContext` can be different in programmatic usage, so it might not have the same context with the gateway because this is just the runtime

Reply:
I made sure that the `transportContext` always has a log, updated the typings too.

Reply:
Ah, have overseen that one. I'll make a todo, thanks for pointing it out!

Reply:
Or maybe just keep the transportContext option? 🤔

Reply:
Or we can have build a default transport context but this might be an extra for this PR.